pbs-rules-set.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards">
+    <description>WooCommerce extension PHP_CodeSniffer ruleset.</description>
+
+    <!-- Exclude paths -->
+    <exclude-pattern>tests/</exclude-pattern>
+    <exclude-pattern>woo-includes/woo-functions.php</exclude-pattern>
+    <exclude-pattern>woo-includes/class-wc-dependencies.php</exclude-pattern>
```

Comment:
**suggestion:** These two lines can be removed because we do not have these classes.